packages/react-router-dev/vite/plugin.ts

```diff
@@ -1646,14 +1646,32 @@ function addRefreshWrapper(
         ]
       : [];
   return (
-    REACT_REFRESH_HEADER.replaceAll("__SOURCE__", JSON.stringify(id)) +
-    code +
-    REACT_REFRESH_FOOTER.replaceAll("__SOURCE__", JSON.stringify(id))
-      .replaceAll("__ACCEPT_EXPORTS__", JSON.stringify(acceptExports))
-      .replaceAll("__ROUTE_ID__", JSON.stringify(route?.id))
+    "\n\n" +
+    withCommentBoundaries(
+      "REACT REFRESH HEADER",
+      REACT_REFRESH_HEADER.replaceAll("__SOURCE__", JSON.stringify(id))
+    ) +
+    "\n\n" +
+    withCommentBoundaries("REACT REFRESH BODY", code) +
```

Comment:
Should this one be left off? When reading the compiled source code, typically I'm looking for this bit — it's only the other blocks I'm trying to ignore, which is where the comments are nice.

Reply:
I don't feel strongly either way. I thought it was nice to give it a name so there's a ubiquitous language for it when debugging or asking users about it, but if you feel its mostly noise I'm ok with removing it.